wasm.md

```diff
@@ -824,6 +827,8 @@ Currently, only one memory may be accessible to a module, and thus the `<memAddr
            )
            ...
          </mems>
+
+    syntax MemId ::= Identifier | Int
```

Comment:
Sort `Index` declared in `data.md` can probably be used here instead.

Reply:
Sorry, the sort to use is `TextFormatIdx`, declared in `data.md`.

Reply:
Thanks! Fixed.